src/en/authors-charm-writing.md

```diff
@@ -337,15 +349,17 @@ cat <<EOF > $vanilla_config
 \$Configuration['Database']['Password'] = '$db_pass';
 EOF
 juju-log "Make the application port available, now that we know we have a site to expose"
+status-set active
 open-port 80
 ```
 
 You will notice that this script uses the backticked command relation-get. This
-is a Juju helper command that fetches the named values from the corresponding
-hook on the service we are connecting to. Usually there will be some indication
-of what these values are, but you can always inspect the corresponding hooks to
-find out. In this case we know that when connected, the MySQL charm will create
-a database and generate random values for things like a username and password.
+is another Juju Hook tool, which in this case fetches the named values from 
+the corresponding hook on the service we are connecting to. Usually there will
+be some indication of what these values are, but you can always inspect the
+corresponding hooks to find out. In this case we know that when connected, the 
+MySQL charm will create a database and generate random values for things like a
```

Comment:
Aha! Yes, I think this usage is arguably correct. However, I will change it and put it in quotes to make it clearer